### ISSUE TITLE
Conv2DTranspose with strides > 1

### ISSUE THREAD

**Issue by cdrose** [google-coral/edgetpu#509]
### Description

I have a model which I am trying to run on an edge TPU, but it performs upsampling of feature maps using layers like:

`layers.Conv2DTranspose(filters=256, kernel_size=2, strides=2, activation='relu')`

These quantize fine and run in the tflite interpreter but do not compile for the edge TPU.

Whats the best way to adjust my model using supported functions to achieve a similar effect?

Are they any similar network structures that run on the edge TPU which I could replicate?

<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Support

### Operating System

Linux

### Coral Device

M.2 Accelerator A+E

### Programming Language

Python 3.9

### Relevant Log Output

_No response_</details>

**Comment by hjonnala** [comment]
@cdrose  Could you please describe the objective of your model and share the CPU tflite model.

**Comment by cdrose** [comment]
@hjonnala The model uses mobilenetv2 as a base model and adds conv2dtranspose layers to upsample and generate segmentation masks from the features. Example attached.

I am currently experimenting with using UpSampling2D followed by Conv2D which seems to work though not yet as well as the original model. This arrangement does compile for edge TPU.

[mobilenet_quant.zip](https://github.com/google-coral/edgetpu/files/7585200/mobilenet_quant.zip)

**Comment by hjonnala** [comment]
@cdrose I am able to compile the model. Are you trying to map all operations to edge TPU?

**Comment by cdrose** [comment]
@hjonnala yes, sorry I should have been more clear. I am aiming to have all operations running on the edge TPU.

**Comment by hjonnala** [comment]
@cdrose do unet models from this [link](https://coral.ai/models/all/#segmentation) would work for you? I Hope you are using the latest `Edge TPU Compiler version 16.0.384591198`.

**Comment by hjonnala** [comment]
Hi @cdrose are you still working on this? This [unet model](https://raw.githubusercontent.com/google-coral/test_data/master/keras_post_training_unet_mv2_128_quant.tflite) has  Conv2DTranspose with strides =2 and those operations are  mapped to edge TPU. Can you try to export the model with fixed tensor shapes and convert to tflite and compile with the edgetpu compiler.

`TRANSPOSE_CONV                 3          Filter, bias, or other param is not constant at compile-time`

**Comment by cdrose** [comment]
Yes I am still working on these models. I have been using the Upsampling2D layer followed by a Conv2D as a workaround. I will try and have a look over the next couple of days and see if I can get it to work.

**Comment by cdrose** [comment]
@hjonnala sorry this took so long but I have finally got some time to do some more work on this issue. I have been working to replicate the example unet architecture above. My models are very similar so this is an ideal example. What I have found is there seems to be some limitations around the Conv2DTranspose operation which I do not fully understand. The two examples that do not work will export to TFLite but will not compile for EdgeTPU, usually with messages about unsupported operations and/or multiple subgraphs.

This works:
```python
    x = layers.Conv2DTranspose(filters, size, strides=2, padding='same')(inputs)
    x = layers.ReLU()(x)
````

![image](https://user-images.githubusercontent.com/4148087/157510081-3113982e-8687-4eb4-bcb5-66dd63d45dec.png)

This does not:
```python
x = layers.Conv2DTranspose(filters, size, strides=2, padding='same', activation='relu')(inputs)
```

![image](https://user-images.githubusercontent.com/4148087/157510383-c624f780-e3cf-4b2f-8ba7-807fa3e7d16d.png)

And this also does not:
```python
x = layers.Conv2DTranspose(filters, size, strides=2, padding='same')(inputs)
x = tf.keras.layers.Dropout(0.2)(x)
x = layers.ReLU()(x)
```

![image](https://user-images.githubusercontent.com/4148087/157510665-6f777988-60cc-40b6-9c61-cca761d3aad1.png)
